File: Funcionario.java
<?php

class Funcionario
{
    public $nome;
    public $salario;
    public $previdencia;

    public function calcularDesconto()
    {
        $desconto = ($this->salario * 0.275) + $this->previdencia;

        return round($desconto, 2);
    }

    public function mostrarDados()
    {
        echo "Nome: " . $this->nome . "<br>";
        echo "Salário: R$ " . $this->salario . "<br>";
        echo "Previdência: R$ " . $this->previdencia . "<br>";
        echo "Desconto: R$ " . $this->calcularDesconto() . "<br><br>";
    }
}

File: trabalho/funcionario.php
<?php

class Funcionario
{
    public $nome;
    public $salario;
    public $previdencia;

    public function __construct($nome, $salario, $previdencia)
    {
        $this->nome = $nome;
        $this->salario = $salario;
        $this->previdencia = $previdencia;
    }

    public function calcularDesconto()
    {
        return round($this->salario * 0.275 + $this->previdencia, 2);
    }

    public function mostrarDesconto()
    {
        echo "O valor do desconto de " . $this->nome . " é " . $this->calcularDesconto() . ".<br>";
    }
}